src/components/CentralContainer/Sidebar/SidebarTabs/SearchTabPanel/ToggleIconButton.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+    IconButton,
+    IconButtonProps,
+    Tooltip,
+} from "@mui/joy";
+
+
+interface ToggleIconButtonProps extends IconButtonProps {
+    isPressed: boolean;
+    text: string;
+    tooltipTitle: string;
+}
+
+/**
+ * An icon button that can visually show icon pressed status.
+ *
+ * @param props
+ * @param props.tooltipTitle
+ * @param props.text
+ * @param props.isPressed
+ * @param props.rest
+ * @return
+ */
+const ToggleIconButton = ({
+    isPressed,
+    text,
+    tooltipTitle,
+    ...rest
+}: ToggleIconButtonProps) => {
+    return (
+        <Tooltip
+            title={tooltipTitle}
+        >
+            <IconButton
```

Comment:
Can we refer to the original code and use a `<span/>` to wrap the `<IconButton/>`? That way the tooltip is not disabled whenever the button is disabled.